workbench/src/renderer/components/InvestTab/index.jsx

```diff
@@ -186,13 +189,34 @@ class InvestTab extends React.Component {
     );
   }
 
+  handleOpenWorkspace(workspace_dir) {
+    if (workspace_dir) {
+      this.openWorkspaceDir(workspace_dir);
+    }
+  }

```

Comment:
Is there a benefit to defining both `handleOpenWorkspace` and `openWorkspaceDir`? As opposed to doing all the logic in one function? It's fairly inconsequential so if you prefer it this way feel free to leave it as-is.

Reply:
Probably not. I think this is a holdover from when `handleOpenWorkspace` also called `shell.showItemInFolder` when the logfile was valid. With that complexity gone, it does look a little silly now.